### PR TITLE
ODSOCS-2029: Adding release notes for oauth token audit logging excep…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…tion

https://issues.redhat.com/browse/OSDOCS-2029

Preview: https://deploy-preview-32068--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-security-log-oauth-tokens

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -57,6 +57,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-security"]
 === Security and compliance
 
+[id="ocp-4-8-security-log-oauth-tokens"]
+==== Audit logging for OAuth access token logout requests
+
+The `Default` audit log policy now logs request bodies for OAuth access token login and logout requests. Previously, only login request bodies were logged.
```
Comment (mburke5678) [inline release_notes/ocp-4-8-release-notes.adoc:63]: @bergerhoffer If I were being picky, I would quibble about _only login request bodies were logged_, as it *could* be interpreted to mean that literally, only requests bodies and nothing else was logged. And suggest: _Previously, logout request bodies were not logged_. 
However, what you have works for release notes.

**Comment by mburke5678** [comment]
@bergerhoffer One (sorta) comment. LGTM

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc:63]: I wasn't super happy with that sentence, so thanks for the good suggestion!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -57,6 +57,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-security"]
 === Security and compliance
 
+[id="ocp-4-8-security-log-oauth-tokens"]
+==== Audit logging for OAuth access token logout requests
+
+The `Default` audit log policy now logs request bodies for OAuth access token login and logout requests. Previously, logout request bodies were not logged.
```
Comment (sttts) [inline release_notes/ocp-4-8-release-notes.adoc:63]: would be more specific:

... now logs request bodies for OAuth access token creation (login) and deletion (logout) requests

Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc]: Thanks, updated. I'll also make this clarification in the other PR (#32053) for pre-4.8 that gets specific about only logging logins.

**Pull request merged**